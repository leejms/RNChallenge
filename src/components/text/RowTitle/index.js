//import liraries
import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';

const Theme = require('../../../styles')

// create a component
const RowTitle = ({children}) => {
    return (
        <View style={styles.container}>
            <Text style={Theme.textStyles.titleText}>{children}</Text>
        </View>
    );
};

// define your styles
const styles = StyleSheet.create({
    container: {
    },
});

//make this component available to the app
export default RowTitle;
