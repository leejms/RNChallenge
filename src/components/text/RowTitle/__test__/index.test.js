import 'react-native'
import React from 'react'
import RowTitle from '../index'

import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <RowTitle>Test</RowTitle>
    ).toJSON()
  expect(tree).toMatchSnapshot()
})