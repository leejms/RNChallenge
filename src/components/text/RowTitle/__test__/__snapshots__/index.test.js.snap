// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={Object {}}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#000000",
        "fontFamily": "Roboto-Regular",
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  >
    Test
  </Text>
</View>
`;
